src/liberror/specification

```diff
@@ -648,3 +648,27 @@ severity:normal
 ingroup:plugin
 module:storage
 see:9 75 109
+
+number:111
+description:Libgcrypt failed to initialize due to a version mismatch.
+severity:normal
+ingroup:plugin
+module:crypto
+
+number:112
+description:Failed to set up the crypto handle correctly.
```

Comment:
that a handle is used is interna.

What about the other initialization errors?